apps/email/js/cards/fld/account_item.html

```diff
@@ -1,4 +1,4 @@
 <a class="fld-account-item">
   <span class="selected-indicator"></span>
-  <span dir="auto" class="fld-account-name"></span>
-</a>
\ No newline at end of file
+  <span class="fld-account-name"></span>
```

Comment:
this is the email adress. `dir="auto"' or `direction: ltr` in the css works equally well. I prefer the last one because:
- it's a pure css solution
- we cannot use start/end css properties with any on them, but if the direction is in the css, it's a lot nearer to any margin/padding... that might be affected, making it more obvious why we can't.